api/controllers/v1/UserController.coffee

```diff
@@ -0,0 +1,29 @@
+# UserController
+#
+# @description :: Server-side logic for managing users
+# @help        :: See http://sailsjs.org/#!/documentation/concepts/Controllers
+
+
+module.exports =
+
+  # `UserController.index()`
+
+  index: (req, res) ->
```

Comment:
Updated all tests to use `res`. I've merged the related PR: https://github.com/DoSomething/quicksilver-api/pull/61